config/database.yml

```diff
@@ -0,0 +1,24 @@
+default: &default
+  adapter: postgis
+  pool: 5
+  timeout: 5000
+  schema_search_path: "public,postgis"
+  script_dir: /usr/local/share/postgis/
+  encoding: unicode
+  username: safecast
+  password:
+
+development:
+  <<: *default
+  database: ingest-solarcast_development
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: ingest-solarcast_test
+
+production:
+  <<: *default
+  database: ingest-solarcast_production
```

Comment:
Beanstalk is gonna have to take at least host & password from env vars. Can you ensure that's supported and commit it here similar to https://github.com/matschaffer/safecastapi/blob/beanstalk-deployment/config/database.yml#L29-L40 (doesn't have to be exactly that, merging default in yaml is 👍  for me)